pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -550,7 +550,10 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 	podUID := string(pod.UID)
 	contName := container.Name
 	allocatedDevicesUpdated := false
-	for k, v := range container.Resources.Limits {
+	// NOTE: Skipping the Resources.Lists is safe here because:
+	// 1. If container Spec mentions Limits only, implicitly Requests, equal to Limits, get added to Spec.
```

Comment:
The commas around `equal to Limits` confused me for a little bit. Not sure how do @ConnorDoyle feel? (guess you're native speak :P